lib/galaxy/jobs/runners/util/cli/job/torque.py

```diff
@@ -83,7 +80,7 @@ def parse_status(self, status, job_ids):
         rval = {}
         for line in status.strip().splitlines():
             try:
-                tree = et.fromstring(line.strip())
+                tree = parse_xml_string(line.strip)
```

Comment:
Did you mean to remove the parens here?

Reply:
Definitely not, thanks!
```suggestion
                tree = parse_xml_string(line.strip())
```